.golangci.yml

```diff
@@ -6,15 +6,28 @@ linters-settings:
     min-confidence: 0
   misspell:
     locale: US
+  govet:
+    check-shadowing: false
+    settings:
+      printf:
+        funcs:
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Infof
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Warnf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Errorf
+          - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
+    disable:
```

Comment:
Can you add a TODO to get rid of those disabled ones?

Reply:
Done!